Interni/NormeProgetto/components/norme-c2.tex

```diff
@@ -0,0 +1,205 @@
+\subsection{Sviluppo}
+\subsubsection{Scopo}
+Il processo di sviluppo ha come scopo il descrivere attività e compiti da svolgere, in modo da realizzare il prodotto finale richiesto dal proponente.
+\subsubsection{Descrizione}
+Il processo di sviluppo è caratterizzato da varie attività,  quali:
+\begin{itemize}
+	\item Analisi dei Requisiti;
+	\item progettazione;
+	\item codifica.
+\end{itemize}
+\subsubsection{Aspettative}
+Le aspettative sono le seguenti:
+\begin{itemize}
+	\item fissare gli obiettivi di sviluppo;
+	\item fissare i vincoli tecnologici;
+	\item fissare i vincoli di design;
+	\item realizzare un prodotto finale che,  oltre a superare i test,  soddisfi i requisiti e le richieste del proponente.
+\end{itemize}
+\subsubsection{Analisi dei requisiti}
+\paragraph{Scopo}
+Gli \anas{} hanno il compito di redigere il documento di \AdR{}, individuando tutti i requisiti che il proponente richiede per la realizzazione del prodotto.  Lo scopo,  quindi,  è quello di:
+\begin{itemize}
+	\item definire lo scopo del lavoro;
+	\item fornire ai \progs{} riferimenti precisi e affidabili;
+	\item fissare le funzionalità e i requisiti concordati col cliente;
+	\item fornire una base per raffinamenti successivi al fine di garantire un miglioramento continuo del prodotto e del processo di sviluppo;
+	item fornire ai \vers{} riferimenti per l'attività di controllo dei test;
+	\item tracciare riferimenti sulla mole di lavoro in modo da avere una stima dei costi.
+\end{itemize}
+\paragraph{Descrizione}
+I requisiti vengono ricavati da diverse fonti,  tra cui:
+\begin{itemize}
+	\item \textbf{Capitolati d'Appalto}: grazie alla lettura approfondita del capitolato scelto vengono individuati dei requisiti;
+	\item \textbf{Casi d'uso}: requisiti estrapolati da uno o più casi d'uso; 
+	\item \textbf{Verbali}: i requisiti possono emergere sia dalle riunioni interne effettuate dagli \anas{}, sia da contatti o incontri con i rappresentanti dell'azienda proponente.
+\end{itemize}
+\paragraph{Aspettative}
+L'obiettivo dell'attività di analisi è la creazione della documentazione formale contenente tutti i requisiti richiesti dal proponente.
+\paragraph{Struttura}
+La struttura che contraddistingue l'\AdR{} è:
+\begin{itemize}
+	\item \textbf{Descrizione}: contiene informazioni riguardanti il prodotto,  le componenti presenti all'interno del sistema,  la piattaforma d'esecuzione e i vincoli di progettazione individuati;
+	item \textbf{Casi d'uso}: vengono identificati gli attori che interagiscono con le componenti del sistema e le interazioni tra sistema,  attori ed elementi esterni.  Approfondimento nel paragrafo §2.2.4.6;
+	\item \textbf{Requisiti}: rappresentati in una tabella che contiene:
+	\begin{itemize}
+ 		\item tracciamento dei requisiti,  suddivisi in obbligatori,  desiderabili e opzionali,  individuati nel sistema durante l'analisi; approfondimento nel paragrafo §2.2.4.5;
```

Comment:
rimuovere doppio spazio prima di "... suddivisi in obbligatori ..."

rimuovere doppio spazio prima di "... desiderabili e opzionali ..."

rimuovere doppio spazio prima di "... individuati nel sistema ..."

Reply:
Fixed